Scripts/install_hyprland.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/env bash
+source global_fn.sh
+if [ $? -ne 0 ]; then
+    echo "Error: unable to source global_fn.sh, please execute from $(dirname $(realpath $0))..."
+    exit 1
+fi
+BldDir="$(dirname $(dirname $(realpath $0)))/Source/build"
+
+hyprland_clone="$HOME/.cache/hyprdots/Hyprland-clone"
+
+chk_aurh
+
+if ! pkg_installed hyprland-gits ; then
```

Comment:
`hyprland-gits` look like a typo here, or is it just to test the condition?

Reply:
oh no Imma fix this. I'm testing this and did not revert stream .